src/views/TransactionsListPage.vue

```diff
@@ -26,22 +26,25 @@ import BaseDropdown from '@/components/BaseDropdown.vue';
 import BaseInnerBlock from '@/components/BaseInnerBlock.vue';
 import BaseCopyRow from '@/components/BaseCopyRow.vue';
 import { ref } from 'vue';
+import { useI18n } from 'vue-i18n';
+
+const { t } = useI18n({ useScope: 'global' });
 
 const tabs = [
-  { label: 'Transactions', value: 'transactions' },
-  { label: 'Transfers', value: 'transfers' },
-  { label: 'Mints', value: 'mints' },
-  { label: 'Burns', value: 'burns' },
-  { label: 'Grants', value: 'grants' },
-  { label: 'Revokes', value: 'revokes' },
+  { label: t('transactions'), value: 'transactions' },
+  { label: t('transfers'), value: 'transfers' },
+  { label: t('mints'), value: 'mints' },
+  { label: t('burns'), value: 'burns' },
+  { label: t('grants'), value: 'grants' },
+  { label: t('revokes'), value: 'revokes' },
 ];
 
 const filterItems = [
-  { label: 'Today', value: 'today' },
-  { label: 'Week', value: 'week' },
-  { label: 'Month', value: 'month' },
-  { label: 'Year', value: 'year' },
-  { label: 'All time', value: 'all-time' },
+  { label: t('time.today'), value: 'today' },
+  { label: t('time.week'), value: 'week' },
+  { label: t('time.month'), value: 'month' },
+  { label: t('time.year'), value: 'year' },
+  { label: t('time.allTime'), value: 'allTime' },
 ];
```

Comment:
Why don't move it to `const` module?

Reply:
I'll do that in the future when I need these filters. so far this page is just like a testing ground for components